### DISCUSSION TITLE
Typst support

### DISCUSSION THREAD

**Discussion by SebVen** [davidgohel/flextable#660] [Ideas]
Quarto has recently [added support](https://quarto.org/docs/output-formats/typst.html) for [Typst](https://typst.app/). Currently, flextables are rendered as images by Typst. It would be great to add full support to `flextable` for Typst as this would save many users hours of work wrestling with LaTeX.

**Comment by jpkrooney** [comment]
Hi. Also wondering if there is a plan to add typst support? Thanks!

**Comment by davidgohel** [comment]
Hi,

Typst is on my radar, it's clearer than LaTeX, which admittedly isn't a very high bar.

That said, flextable already speaks HTML, Word, PowerPoint, PDF, grid graphics and RTF. Adding a new language to the family is not a weekend project. And so far, the queue of people asking for Typst support is you two ;)

So: are you using Typst for an actual project where flextable is a blocker, or is this more of a "I'd love this to exist" situation?

**Comment by jpkrooney** [comment]
Hi David,
Yes for sure flextable already does alot, and thank you very much for a fantastic package!
For me I only recently heard of typst, and yes the clarity over LaTeX is an appeal, but another important consideration here is speed! A full set of TLFs for a clinical trial can take hours to render via LaTeX, which makes "one small final edit" into a time and CPU expensive task. Typst offers the promise of reducing/eliminating this cost! So no, I don't have an actual project right now where I'm using Typst currently, more I'm trying to see what are the blockers to using Typst in my work flow going forward, and was just wondering what your plan is about it.
Thanks,
James

**Comment by JTCDCTWB** [comment]
Hi, following up on #660. Flextable is extremely excellent, and Typst’s advantages are also very obvious. We are glad to see that flextable has improved cross-referencing support in Quarto in recent versions, and we sincerely hope that you can follow up with full support for Typst. This will help us fully adopt Typst in production workflows and solve the current problem of flextable being rendered as static images in Typst. Would love to know your thoughts on this. Thanks for the amazing package!

**Comment by davidgohel** [comment]
implemented since version `0.10.0` now available on CRAN !
